onnx2torch/node_converters/mod.py

```diff
@@ -0,0 +1,27 @@
+__all__ = [
+    'OnnxMod',
+]
+
+import torch
+from torch import nn
+
+from onnx2torch.node_converters.registry import add_converter
+from onnx2torch.onnx_graph import OnnxGraph
+from onnx2torch.onnx_node import OnnxNode
+from onnx2torch.utils.common import OnnxToTorchModule
+from onnx2torch.utils.common import OperationConverterResult
+from onnx2torch.utils.common import onnx_mapping_from_node
+
+
+class OnnxMod(nn.Module, OnnxToTorchModule):  # pylint: disable=missing-class-docstring
```

Comment:
ONNX [Mod](https://github.com/onnx/onnx/blob/main/docs/Operators.md#Mod) operation has fmod attribute:
```
Attributes

fmod : int (default is 0)
    Whether the operator should behave like fmod (default=0 meaning it will do integer mods); Set this to 1 to force fmod treatment
```

It must be taken into account.

Reply:
I added the fmod attribute.

By the way, I mentioned the problems I encountered joining it, but of course I have solved them.

When `fmod` is 1, the mod could mod float. `fmod` is 0, the Mod only handle integers.